components/metrics/WebsiteChart.js

```diff
@@ -7,47 +7,39 @@ import DateFilter from 'components/common/DateFilter';
 import StickyHeader from 'components/helpers/StickyHeader';
 import useFetch from 'hooks/useFetch';
 import useDateRange from 'hooks/useDateRange';
-import useTimezone from 'hooks/useTimezone';
 import usePageQuery from 'hooks/usePageQuery';
-import { getDateArray, getDateLength } from 'lib/date';
 import ErrorMessage from 'components/common/ErrorMessage';
 import FilterTags from 'components/metrics/FilterTags';
-import useShareToken from 'hooks/useShareToken';
-import { TOKEN_HEADER } from 'lib/constants';
+import { getDateArray, getDateLength } from 'lib/date';
 import styles from './WebsiteChart.module.css';
 
 export default function WebsiteChart({
   websiteId,
   //title,
-  //domain,
+  domain,
   stickyHeader = false,
   //showLink = false,
   hideChart = false,
   onDataLoad = () => {},
 }) {
-  const shareToken = useShareToken();
   const [dateRange, setDateRange] = useDateRange(websiteId);
   const { startDate, endDate, unit, value, modified } = dateRange;
-  const [timezone] = useTimezone();
```

Comment:
We now show data in UTC (that may be fine, not sure yet) but the the last time in the hourly chart is always in the future and thus will always be empty, which is weird.

![image](https://github.com/Hashnode/umami/assets/7524745/21539231-e898-4372-b507-95a61d02d0b1)

Reply:
This issue becomes more obvious if the difference to UTC gets bigger. Check this screenshot emulating `Asia/Tokyo` (GMT+09:00)

![image](https://github.com/Hashnode/umami/assets/7524745/df14a5ca-0ed2-4646-a73c-1a2d42ee02e6)

Reply:
Thought: we may want to use the timezone here and construct the dates in the APIs by taking the tz into account?

Reply:
How are you making these screenshots?

Reply:
https://shottr.cc
It's really cool and free.

Reply:
Nice, this works now. But there is one weird thing for "This year" 👇🏼 It is not showing any views for January.
![image](https://github.com/Hashnode/umami/assets/7524745/1e04e5d1-5295-41f8-92fa-a38807fc661d)

https://hn-analytics-umami-git-shad-use-gql-hashnode-prod.vercel.app/share/o0oyhroj/4cc6a462-e4cc-4115-b69d-dfaec9683f4b?view=country